app/utils.js

```diff
@@ -422,6 +427,27 @@ function refreshExchangeRates() {
 			}
 		});
 	}
+
+	if (coins[config.coin].exchangeRateDataBTCEUR) {
+		request(coins[config.coin].exchangeRateDataBTCEUR.jsonUrl, function(error, response, body) {
+			if (error == null && response && response.statusCode && response.statusCode == 200) {
+				var responseBody = JSON.parse(body);
+				var exchangeRate = coins[config.coin].exchangeRateDataBTCEUR.responseBodySelectorFunction(responseBody);
+				if (exchangeRate != null) {
+					if (global.exchangeRates === undefined) global.exchangeRates = {};
+					global.exchangeRates['btceur'] = exchangeRate;
+					global.exchangeRatesUpdateTime = new Date();
+
+					debugLog("Using exchange rates: " + JSON.stringify(global.exchangeRates) + " starting at " + global.exchangeRatesUpdateTime);
+
+				} else {
+					debugLog("Unable to get exchange rate data");
+				}
+			} else {
+				logError("3825isdgij", {error:error, response:response, body:body});
```

Comment:
3825isdgij what is this?

Reply:
dunno, it's used in other places so I've did it like it was

Reply:
looks like a random string :D usually, the first argument is the name of the module so should be logError("utils", {error:error, response:response, body:body}); check if this is the case in logError function